nabs/utils.py

```diff
@@ -110,3 +112,38 @@ def add_named_kwargs_to_signature(
     )
 
     return sig.replace(parameters=start_params + wrapper_params + end_params)
+
+
+def orange(start, stop, num, rtol=1.e-5, atol=1.e-7):
```

Comment:
It took me a very long time to realize this was probably "o-range" and not the fruit

Reply:
This method name was my favorite part of the PR, honestly ;D